test/topology/test_topology.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test consistency of schema changes with topology changes.
+"""
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_add_server_add_column(manager, random_tables):
+    """Add a node and then add a column to a table and verify"""
+    table = await random_tables.add_table(ncolumns=5)
+    await manager.server_add()
+    await table.add_column()
+    await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+async def test_stop_server_add_column(manager, random_tables):
+    """Add a node, stop an original node, add a column"""
+    servers = await manager.servers()
+    table = await random_tables.add_table(ncolumns=5)
+    await manager.server_add()
+    await manager.server_stop(servers[1])
```

Comment:
Is it guaranteed that manager.servers() always begins with 3 nodes?

Reply:
It's specified in `suite.yaml`
https://github.com/alecco/scylladb/blob/raft-test-topology-part-3/test/topology/suite.yaml#L2